src/rgw/rgw_process.cc

```diff
@@ -291,6 +291,8 @@ int process_request(rgw::sal::RGWRadosStore* const store,
   } catch (rgw::io::Exception& e) {
     dout(0) << "ERROR: client_io->complete_request() returned "
             << e.what() << dendl;
+    perfcounter->inc(l_rgw_qlen, -1);
+    perfcounter->inc(l_rgw_qactive, -1);
```

Comment:
can we address this issue in a separate pr? the counters should be encapsulated by the frontends, and shouldn't leak into process_request()

Reply:
Done & working on handling it at the frontend level.

Reply:
hi @mkogan1 , is the pr fix perfcounter->inc(l_rgw_qlen, -1); already submit? i search no result for it, if have not summit, i would like to submit a new pr about it

Reply:
@joke-lee It's not merged yet, waiting on PR #36428 to see if it will facilitate a better solution than directly modifying counters in precess_request